### ISSUE TITLE
ALMA-Fix: Set up productive ALMA-Fix Index workflow at test.lobid.org/ressources

### ISSUE THREAD

**Issue by TobiasNx** [hbz/lobid-resources#1435]

**Comment by TobiasNx** [comment]
@hagbeck: To give you a heads up. We are about to have a better ALMA-Transformation with Fix that we would like to test with you so that we can switch it to the productive system.

**Comment by dr0i** [comment]
As mentioned today at standup, this is not ready yet. The index `resources-alma-fix` was build with morph due to a configuration failure.

**Comment by TobiasNx** [comment]
I didn't say it is ready ;-)

**Comment by dr0i** [comment]
I removed the "productive" from the title, as we have the Alma Morph ETL in parallel for now, which is triggered by a call to the webhook listener. For "load level" reasons we don't want  ETL of Morph and Fix in parallel (both access the same dump and the same ES cluster), and as the webhook caller of the invoked ETL can atm only call _one_ other webhook listener (Dortmund) to inform when ETL is finished, the ETL of Alma Fix is triggered by cron (in opposite to a productive workflow where the webhook listener awaits a call).
Cron is scheduled at  `Sunday 12:00` for basedump and at `Everyday at 10:00` for updates.

To make this "productive" we should make use of the call to the Webhook listener:
- [ ] make webhook caller able to inform multiple endpoints when finished ("chaining")
- [ ] wait until Alma Morph is discarded in total and switch to Alma Fix

I like the first approach.  We would have to open a new issue for that.

**Comment by dr0i** [comment]
@UBmakla you can test this now. The API URL is https://test.lobid.org/resources/ and you can use this like the one you already use (https://alma.lobid.org/resources/99371678351606441.json), e.g. https://test.lobid.org/resources/99371678351606441.json.
We are happy if you open new issues here if you discover something not yet perfect!

**Comment by UBmakla** [comment]
Thanks for the information, I look forward to the test. If I have comments, should they be attached to this ticket or should I create a new one?

**Comment by dr0i** [comment]
@UBmakla please open new issues, so we can track these concisely and be focused.
 If you feel there is something that is not worth for the public or whatever you can also write us an email :)
